tests/testthat/test-helpers.R

```diff
@@ -25,3 +25,33 @@ test_that("h_all_equivalent returns FALSE for non-equivalent objects", {
   # Mean relative difference: 0.2 > tolerance = 0.1
   expect_false(result)
 })
+
+# h_plot_data_df ----
+
+test_that("h_plot_data_df valid object for sample Data object with placebo and no blinding", {
+  data <- h_get_data()
+  result <- h_plot_data_df(data)
+  expected <- data.frame(
+    patient = 1:12,
+    ID = paste(" ", data@ID),
+    cohort = data@cohort,
+    dose = ifelse(data@x == data@doseGrid[1], 0, data@x),
+    toxicity = as.factor(data@y)
+  )
```

Comment:
it is better to give the explicit values instead of getting them out of `data` - so we can actually see the contents